test/gtest/ucp/test_ucp_worker.cc

```diff
@@ -29,6 +29,12 @@ class test_ucp_worker_discard : public ucp_test {
         std::vector<uct_pending_req_t*>    pending_reqs;
         unsigned                           flush_count;
         unsigned                           pending_add_count;
+
+        ep_test_info_t() {
+            pending_reqs.clear();
```

Comment:
not needed, std::vector has its own constructor.

Reply:
done